packages/database/schema.prisma

```diff
@@ -319,19 +319,21 @@ enum WidgetPlacement {
 }
 
 model Product {
-  id                  String          @id @default(cuid())
-  createdAt           DateTime        @default(now()) @map(name: "created_at")
-  updatedAt           DateTime        @updatedAt @map(name: "updated_at")
-  name                String
-  team                Team            @relation(fields: [teamId], references: [id], onDelete: Cascade)
-  teamId              String
-  environments        Environment[]
-  brandColor          String          @default("#64748b")
-  recontactDays       Int             @default(7)
-  formbricksSignature Boolean         @default(true)
-  placement           WidgetPlacement @default(bottomRight)
-  clickOutsideClose   Boolean         @default(true)
-  darkOverlay         Boolean         @default(false)
+  id                   String          @id @default(cuid())
+  createdAt            DateTime        @default(now()) @map(name: "created_at")
+  updatedAt            DateTime        @updatedAt @map(name: "updated_at")
+  name                 String
+  team                 Team            @relation(fields: [teamId], references: [id], onDelete: Cascade)
+  teamId               String
+  environments         Environment[]
+  brandColor           String          @default("#64748b")
+  showHighlightBorder  Boolean         @default(false)
```

Comment:
please only use one database variable for the highlightBorderColor, e.g.:
```
highlightBorderColor String?
```
and set it to null if no highlight border color is set.

We want to avoid having too many database variables and also it feels inconsistent in the data to have `showHighlightBorder` set to `false` and still have a `highlightBorderColor` set to a specific value.